src/main/java/org/folio/des/config/FolioExecutionContextHelper.java

```diff
@@ -53,31 +44,24 @@ public boolean isModuleRegistered() {
   }
 
   public FolioExecutionContext getFolioExecutionContext(String tenantId) {
+    Map<String, Collection<String>> tenantOkapiHeaders = new ConcurrentHashMap<>();
```

Comment:
Why not just HashMap? I don't see usage of this variable from different threads

Reply:
Thank you, replaced.